src/main/webapp/index.html

```diff
@@ -4,15 +4,16 @@
     <meta charset="UTF-8">
     <title>SAY</title>
     <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="script.js"></script>
   </head>
   <body>  
     <div>
-      <form>
-        <p>Enter a YouTube video link:</p>
-        <textarea name="link-input"></textarea>
+      <form onsubmit="return false">
```

Comment:
Is there a reason why you set the onsubmit here?

Reply:
The charts API can be configured to show the chart everytime the page loads or everytime a button is clicked, but not both. Since we want to have it only after the button is clicked, but submitting the form reloads the page, setting onsubmit here allows us to show the chart only after the form is submitted.